.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,53 +1,28 @@
 ## Description
 
-<!--- 
-Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
-For the benefit of the community, please do not assume prior context.
-
-Provide details that support your chosen implementation, including: 
-- breaking changes
-- alternatives considered
-- changes to the API
-- demos (screenshots, videos) if you find that useful
-- etc.
--->
-
-
-## References
-
-<!--- 
-Include any links supporting this change such as a:
-
-- GitHub Issue/PR number addressed or fixed
-- Auth0 Community post
-- StackOverflow post
-- Support forum thread
-- Related pull requests/issues from other repos
-
-If there are no references, simply delete this section.
--->
+<!-- Describe the purpose of this PR along with any background information and the impacts of the proposed change. 
+For the benefit of the community, please do not assume prior context. -->
 
+## Checklist
 
-## Testing
+**Note:** Checklist required to be completed before a PR is considered to be reviewable.
 
-<!--- 
-Describe how this can be tested by reviewers. Be specific about anything not tested and reasons why. 
-If this library has unit and/or integration testing, tests should be added for new functionality and 
-existing tests should complete without errors.
--->
+#### Auth0 Code of Conduct
+- [ ] I have read and agreed to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)
 
-- [ ] This change adds test coverage for new/changed/fixed functionality
```

Comment:
The biggest missing piece of the current PR template, IMO, is the lack of a checkbox that explicitly states that tests are not required. Otherwise it isn't clear why this box is unchecked.